### PR TITLE
ENVIRONMENT needs to be an env var not a build arg for ci

### PR DESCRIPTION
**Pull request by j16r**
Also cleaned up some redundant vars from the ci image.

### CODE REVIEW

File: internal/ci/Dockerfile [hunk 1]
```diff
@@ -4,11 +4,6 @@ FROM smartcontract/builder:1.0.1
 ENV DEBIAN_FRONTEND noninteractive
 RUN apt-get update && apt-get install -y jq
 
-# Have to reintroduce ENV vars from Baidu's SGX SDK image
-ENV PATH /root/.cargo/bin:/go/bin:/usr/local/go/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/opt/sgxsdk/bin:/opt/sgxsdk/bin/x64
```
Comment (rupurt) [inline internal/ci/Dockerfile]: Is this related to the environment change? Or just a general cleanup?

I don't see how it relates to the environment variable for `development`

Comment (j16r) [reply internal/ci/Dockerfile]: Just a cleanup
